src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -39,15 +39,18 @@
     /**
      * Loads one or several .env files.
      *
-     * @param ...string A list of files to load
+     * @param $path A file to load
+     * @param ...string A list of additionnal files to load
```

Comment:
not true. ``string[]`` means the outside is passing an array. Here, the signature is not an array but a variadic arg